types/src/Experiment/enums.ts

```diff
@@ -170,6 +170,11 @@ export enum SEGMENT_STATUS {
   UNLOCKED = 'Unlocked',
 }
 
+export enum STRATIFICATION_FACTOR_STATUS {
```

Comment:
@ppratikcr7 i think this just needs removed now if it's now unused right? Looks good to me otherwise, can approve and merge after that

Reply:
Done